Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -270,9 +265,6 @@ extension BrowserViewController: WKNavigationDelegate {
         
         // Once we check the redirect chain only need the last (final) url from our redirect chain
         if let redirectURL = redirectChain.last?.url {
-          // Cancel the original request. We don't want it to load as it's tracking us
-          decisionHandler(.cancel, preferences)
```

Comment:
Same here

Reply:
This one I tested it was working correctly